meltano/cfp-pipeline/meltano.yml

```diff
@@ -0,0 +1,59 @@
+version: 1
+send_anonymous_usage_stats: true
+project_id: 45da6c57-af78-4ac6-8e0c-d26e914bb9e5
+plugins:
+  extractors:
+  - name: tap-github
+    namespace: tap_github
+    variant: singer-io
+    pip_url: tap-github
+    executable: tap-github
+    capabilities:
+    - properties
+    - discover
+    - state
+    config:
+      access_token: $TAP_GITHUB_ACCESS_TOKEN
+      repository: CodeForPhilly/cfp-data-pipeline
+  - name: tap-slack
+    namespace: tap_slack
+    variant: singer-io
+    pip_url: tap-slack
+    executable: tap-slack
+    capabilities:
+    - catalog
+    - discover
+    - state
+    config:
+      token: $TAP_SLACK_TOKEN
+      start_date: '2021-01-08T00:00:00'
+      private_channels: false
+      join_public_channels: true
+      exclude_archived: true
+  loaders:
+  - name: target-csv
+    variant: singer-io
+    pip_url: target-csv
+  - name: target-postgres
+    variant: datamill-co
+    pip_url: singer-target-postgres
+    config:
+      postgres_host: postgres
+      postgres_port: 5432
+      postgres_database: airflow
+      postgres_username: $TARGET_POSTGRES_POSTGRES_USERNAME
+      postgres_password: $TARGET_POSTGRES_POSTGRES_PASSWORD
+      postgres_schema: public
```

Comment:
Our postgres database should be `datawarehouse` and the schema should be `views`. The airflow database is specifically for Ariflow's Metadata Database.